src/main/java/frc/robot/subsystems/swerve/SwerveSubsystem.java

```diff
@@ -476,7 +478,7 @@ private void updateVision() {
         camera.setSimPose(estPose, camera, newResult);
         Logger.recordOutput("Vision/Vision Pose From " + camera.getName(), visionPose);
         Logger.recordOutput("Vision/Vision Pose2d From " + camera.getName(), visionPose.toPose2d());
-        pose = pose.interpolate(visionPose.toPose2d(), 0.25);
+        // pose = pose.interpolate(visionPose.toPose2d(), 0.25);
```

Comment:
Delete vs comment out? Do we never want to interpolate the pose?

Reply:
from my understanding this was a temp fix before we realized that the timestamps were not synced. think this can be deleted